pages/02.install/04.macos/default.md

```diff
@@ -3,14 +3,37 @@ title: macOS
 ---
 
 ### Introduction
-! No one on the dev team owns a Mac. These instructions are suggested by our users. If you think you can improve them, please submit a PR.
+! No one on the dev team owns a Mac. These instructions are suggested by our users. If you think you can improve them, please [edit this wiki](https://github.com/Kareadita/Wiki/edit/main/pages/02.install/04.macos/default.md) and submit a PR.
+
+1. [Download Kavita for macOS](https://www.kavitareader.com/#downloads-v1-macos). The file you want is "kavita-osx-x64.tar.gz".
+
+2. Unpack Kavita to your User Folder or folder of choice. For example, in Terminal:
+    * `cd ~`
+    * `mv ~/Downloads/kavita-osx-x64.tar.gz ~`
+    * `tar -xf kavita-osx-x64.tar.gz`
 
-1. [Download Kavita for macOS](https://www.kavitareader.com/#downloads-v1-macos)
-2. Unpack Kavita to your User Folder or folder of choice.
 3. Make Kavita executable.
-    * In Terminal: `sudo chmod +x ~/Kavita/Kavita`
+    * `sudo chmod +x ~/Kavita/Kavita`
+
 4. Disable Gatekeeper for Kavita's folder so that Kavita and all .dll and .dylib files can be opened
-    * In Terminal: `attrx -r -d com.apple.quarantine ~/Downloads/Kavita`
-5. Start Kavita with right-click -> Open (or in Terminal: `~/Kavita/Kavita` )
+    * `xattr -r -d com.apple.quarantine ~/Kavita/`
+
+5. Help Kavita find sqlite
+    * `export alias sqlite=sqlite3`
+
+6. Start Kavita in Terminal. You need to launch Kavita from within the `~/Kavita` folder, or else `appsettings.json` will fail to be acquired.
+    * `cd ~/Kavita`
+    * `./Kavita &`
```

Comment:
Not sure about using `&`.  I personally discourage using it unless needed because manually managing demons is a pain.
We should just link to the linux service setup as it was removed.

Reply:
Sure thing, `&` removed. But the linux service setup is not applicable to Mac. Linux has `systemd`/`systemctl`, but MacOS has `launchd`/`launchctl` and the execution is controlled with a `plist` file... it looks doable but tricky

Reply:
Could we just copy paste and modify? Or they simply differ too much?

Reply:
Too different unfortunately. The `plist` for MacOS is a special XML format

Reply:
That is true, the plist is very different. Let's for now leave the Linux section off.

Reply:
Is there no way to have something similar?
If not we might be able to add `&` back as an optional parameter explaining what it does i guess. 🤔